components/Timeline.tsx

```diff
@@ -0,0 +1,112 @@
+import Image from 'next/image';
+
+function Phase({ title, date, duration, imageSrc, phaseText }) {
+    return (
+        <div className="relative flex justify-between">
+            <div className="flex flex-col items-center w-10 mr-4 md:w-24">
+                <div>
+                    <div className="flex items-center justify-center w-8 h-8 bg-blue-200 rounded-full dark:bg-gray-600">
+                        <div className="w-4 h-4 bg-blue-600 rounded-full dark:bg-blue-400"></div>
+                    </div>
+                </div>
+                <div className="w-px h-full bg-blue-300 dark:bg-gray-600"></div>
+            </div>
+            <div>
+                <h2 className="inline-block px-4 py-2 mb-4 text-xs font-medium text-gray-100 bg-gradient-to-r from-blue-500 to-blue-900 dark:from-blue-400 dark:to-blue-500 rounded-3xl dark:text-gray-100">
+                    {date} <br />
+                    ({duration})
+                </h2>
+                <div className="relative flex-1 mb-10 bg-white border-b-4 border-blue-200 shadow dark:border-gray-700 rounded-3xl dark:bg-gray-900">
+                    <img
+                        src={imageSrc}
+                        alt={title}
+                        className="w-full mb-6"
+                    />
+                    <div className="relative z-20 p-6">
+
+                        <p className="mb-2 text-xl font-bold text-gray-600 dark:text-gray-400">
+                            {title}
+                        </p>
+                        <p className="text-gray-700 dark:text-gray-500">
+                            {phaseText}
+                        </p>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+
+
+export default function Timeline() {
+    return (
+        <div className="justify-center max-w-6xl lg:py-8 md:px-6 z-0">
+            <div className="w-full mx-auto lg:max-w-3xl">
+                <Phase
+                    title="Formation and Exploration"
+                    date="2nd October - 15th October 2023"
+                    duration="2 Weeks"
+                    imageSrc="/assets/e-lab/phases/co-founder.jpg"
```

Comment:
Image src seems to be broken in the preview deployment

Reply:
Where do you see that it is broken, for me it looked fine, where could I check?

Reply:
https://website-nextjs-git-venture-timon-softdevtumai.vercel.app/e-lab